src/Engines/nativeEngine.ts

```diff
@@ -884,6 +935,59 @@ export class NativeEngine extends Engine {
 
         // Shader processor
         this._shaderProcessor = new NativeShaderProcessor();
+
+        if (!this.originBottomLeft) {
+            // prepare YFlip shader and vertex stream
+            Effect.ShadersStore["fullscreenFlipVertexShader"] = `
+                attribute vec2 position;
+                attribute vec2 uv;
+                varying vec2 vUV;
+                void main(void) {
+                    gl_Position = vec4(position, 0.5, 1.0);
+                    vUV = position * 0.5 + 0.5;
+                    vUV.y = 1.0 - vUV.y;
+                }`;
+            Effect.ShadersStore["fullscreenFlipFragmentShader"] = `
+                varying vec2 vUV;
+                uniform sampler2D backBuffer;
+                void main(void)
+                {
+                    vec3 color = texture2D(backBuffer, vUV).xyz;
+                    gl_FragColor = vec4(color, 1.0);

```

Comment:
The backBuffer can't have some alpha != 1?

Reply:
It won't be blended. It's a pure color copy. Alpha has no use in that case.